imports/plugins/core/orders/client/components/OrderCardFulfillmentGroups.js

```diff
@@ -0,0 +1,249 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Mutation } from "react-apollo";
+import withStyles from "@material-ui/core/styles/withStyles";
+import Button from "@material-ui/core/Button";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Divider from "@material-ui/core/Divider";
+import Grid from "@material-ui/core/Grid";
+import Hidden from "@material-ui/core/Hidden";
+import Typography from "@material-ui/core/Typography";
+import Address from "@reactioncommerce/components/Address/v1";
+import { i18next, Reaction } from "/client/api";
+import ConfirmButton from "/imports/client/ui/components/ConfirmButton";
+import cancelOrderItemMutation from "../graphql/mutations/cancelOrderItem";
+import OrderCardFulfillmentGroupItem from "./OrderCardFulfillmentGroupItem";
+import OrderCardFulfillmentGroupTrackingNumber from "./OrderCardFulfillmentGroupTrackingNumber";
+import OrderCardFulfillmentGroupStatusButton from "./OrderCardFulfillmentGroupStatusButton";
+import OrderStatusChip from "./OrderStatusChip";
+
+const styles = (theme) => ({
+  fulfillmentGroupHeader: {
+    marginBottom: theme.spacing.unit * 4
+  },
+  verticalDivider: {
+    backgroundColor: "#e6e6e6",
```

Comment:
is there a color in the theme that can be used here? Let's stay away from one off styles.

Reply:
Fixed